src/markdown/downloads/disbursements.md

```diff
@@ -24,65 +24,39 @@ Download fiscal year data:
+* _State_ Funds disbursed to states fall under the jurisdiction of each state, and each state determines how the funds will be used.
+* _State 8(g)_ 27% of revenue from leases in the 8(g) Zone (the first three nautical miles of the Outer Continental Shelf) are shared with states.
+* _State GoMESA_ 37.5% of revenue from certain leases in the Gulf of Mexico are shared with Alabama, Louisiana, Mississippi, and Texas.
+* _Other_ Certain funds are directed back to the federal agencies that administer these lands (e.g., BLM, U.S. Fish and Wildlife Service, and U.S. Forest Service) to help cover the agencies’ operational costs. The Ultra-Deepwater Research Program and the Mescal Settlement Agreement also receive $50 million each.
 
-_Reclamation_ Established by Congress in 1902 to pay for Bureau of Reclamation projects, this fund supports the establishment of critical infrastructure projects like dams and power plants.
+_Onshore/Offshore_ Whether the origin of the revenue being disbursed is offshore or onshore.
 
-_American Indian Tribes_ ONRR disburses 100% of revenue collected from resource extraction on American Indian lands back to the Indian tribes and individual Indian landowners.
+_State_ The state that received the disbursement in accordance with revenue-sharing legislation.
 
-_Other_ Certain onshore funds are directed back to the federal agencies that administer these lands (e.g., BLM, U.S. Fish and Wildlife Service, and U.S. Forest Service) to help cover the agencies’ operational costs. The Ultra-Deepwater Research Program and the Mescal Settlement Agreement also receive $50 million each.
+_County_ The county that received the disbursement in accordance with revenue-sharing legislation.
 
-### Offshore
-
-_U.S. Treasury_ The majority of offshore revenue is disbursed to the Treasury, which enters it into the General Fund, the federal government’s basic operating fund. The General Fund pays for roughly two-thirds of all federal expenditures, including the U.S. military, national parks, and schools.
-
-_Land and Water Conservation Fund_ This fund provides matching grants to states and local governments to buy and develop public outdoor recreation areas across the 50 states.
-
-_Historic Preservation Fund_ This fund helps preserve U.S. historical and archaeological sites and cultural heritage through grants to state and tribal historic preservation offices.
-
-_States_ States receive federal Outer Continental Shelf revenue in two ways:
-
-- 27% of revenue from leases in the 8(g) Zone (the first three nautical miles of the Outer Continental Shelf) are shared with states.
-- 37.5% of revenue from certain leases in the Gulf of Mexico are shared with Alabama, Louisiana, Mississippi, and Texas.
-
-_Other_ Certain offshore funds are directed back to the federal agencies that administer these lands (e.g., BOEM and BSEE) to help cover the agencies’ operational costs.
-
-## Archive
```

Comment:
We should consider moving this data to the new archive. We can do that later, though.